index.js

```diff
@@ -1,6 +1,6 @@
 const path = require('path');
 
-const include_dir = path.relative('.', __dirname);
+const include_dir = `"path.relative('.', __dirname)"`;
```

Comment:
```suggestion
const include_dir = `"${path.relative('.', __dirname)}"`;
```
Isn't this the intended change according to https://github.com/nodejs/node-addon-examples/issues/183#issuecomment-836050243?

Reply:
You are correct. I meant to do that (must've been too tired 😅). Thanks for catching that